source/docs/guides/secure-wordpress-or-drupal-with-sucuri.md

```diff
@@ -0,0 +1,89 @@
+Whether you're looking to secure a personal site or an enterprise client, Sucuri has you covered with 24/7 ticket support and incredible response times.
+
+## Prerequisites
+
+### Direct Your Domain to Pantheon
+In order to add Sucuri Cloud Proxy services, you must already have a domain directed to Pantheon and added to the Live environment. For more instructions, please see [DNS Records for Directing Your Domain to Your Pantheon Site](/source/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site/).
+
+## Create a Sucuri Account
+
+Get started by [selecting a plan](https://sucuri.net/website-firewall/signup) that fits your needs. Once a plan is selected, you will be instructed to create your Sucuri account and select a payment method.
+
+![Sucuri sign up form](/source/docs/assets/images/sucuri-cloud-proxy-signup.png)​
+
+## Login to your Sucuri Account
+
+After receiving your confirmation email, sign in to your [Sucuri account](https://login.sucuri.net/login/). From within the Dashboard, click on **CloudProxy Website Firewall**.
+
+![Sucuri Account Settings](/source/docs/assets/images/sucuri-account-settings.png)
+
+## Add Your Site
+
+Enter your domain and click **Add Site** to enable CloudProxy. You can add it with or without www, Sucuri will remove the www automatically.
+
+![Sucuri Add Site](/source/docs/assets/images/sucuri-add-website.png)
+
+## Verify Domain Name
+
+You will see a pop-up that asks you to verify the domain name and server location. Pantheon sites are served from North America, but you can select a location that represents where you live.
+
+Indicate whether you are currently under a DDoS attack or if you would like to restrict your site's backend to whitelisted IPs. When you are ready, click **Proceed**
+
+![Sucuri Add New Domain](/source/docs/assets/images/sucuri-add-new-domain.png)
+
+CloudProxy will display a status bar which will configure the account settings for your domain name. It should not take more than 1-2 minutes and will close automatically when finished.
+
+![Sucuri CloudProxy Loading](/source/docs/assets/images/sucuri-cloud-proxy-loading.png)
+
+## Activating CloudProxy
+
+When the status bar is done, you will see a long page of information. Follow the instructions within the blue area labeled **Activating CloudProxy**:
+
+1. First, check the URL provided and verify your site loads on the temporary address.
+2. Second, if you need SSL support click the link provided for further instructions.
+3. Login to where your DNS settings are for your domain name and remove the records you received from Pantheon. Add just an **A Record** to the IP address Sucuri lists. Save your DNS settings and come back to your Sucuri dashboard screen.
```

Comment:
@rachelwhitton The only records the user will need to remove is the CNAME which is outlined in the instructions. They just need to update the A record on their DNS settings for the production domain since the CNAME name will be added in Sucuri.